changelog/unreleased/issue-340

```diff
@@ -0,0 +1,8 @@
+Enhancement: Add support for Volume Shadow Copy Service (VSS) on Windows
+
+Volume Shadow Copy Service allows read access to files that are locked by
+another process using an exclusive lock. Restic was unable to backup those
+files before. This update enables backing up these files.
```

Comment:
Might want to make the last sentence something like "This update enables backing up these files using the --use-fs-snapshots option", just so users get a quick reference to what they want to read/learn more about.

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/dc44ea2b3cab883d6e8d2bf9d357b3cb784f8b0f